modules/nw-network-flows-object.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/tracking-network-flows.adoc
+
+[id="nw-network-flows-object_{context}"]
+= Network object configuration for tracking network flows
+
+The following YAML shows the configuration fields for network flows collectors in the Cluster Network Operator configuration.
+
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  exportNetworkFlows: <.>
+    netFlow:
+      collectors:
+        - <netflow-collector-1>:2056
```

Comment:
I think we're using underscores (_) for our replaceable variables.

Reply:
Y